input/templates/partials/catalog/reserve/initialize-map-scripts.hbs

```diff
@@ -0,0 +1,79 @@
+<script>
+  $(window).load(function() {
```

Comment:
because this is HTML injected script. It shouldn't be a dependency as google recommends [here](https://developers.google.com/maps/documentation/javascript/tutorial)